lib/resources.yml

```diff
@@ -63,11 +63,13 @@ js:
       - angular2: '*'
     React (latest):
       - react: '*'
+      - react-dom: '*'
```

Comment:
If possible, please use an exact version number here. That avoids unexpected CI breaks.